src/main/java/edu/greenblitz/robotName/subsystems/swerve/Modules/mk4iSwerveModule/MK4ISwerveModule.java

```diff
@@ -115,13 +106,13 @@ public void updateInputs(SwerveModuleInputsAutoLogged inputs) {
         inputs.angularCurrent = angularMotor.getStatorCurrent();
 
         inputs.linearMetersPassed = Conversions.MK4IConversions.convertTicksToMeters(linearMotor.getSelectedSensorPosition());
-        inputs.angularPositionInRads = Conversions.MK4IConversions.convertTicksToRads(angularMotor.getSelectedSensorPosition());
+        inputs.angularPositionRadians = Conversions.MK4IConversions.convertTicksToRadians(angularMotor.getSelectedSensorPosition());
 
-        if (Double.isNaN(Units.degreesToRadians(canCoder.getAbsolutePosition().getValue()))){
+        if (Double.isNaN(Units.degreesToRadians(canCoder.getAbsolutePosition()))){
             inputs.absoluteEncoderPosition = 0;
         }else{
-            inputs.absoluteEncoderPosition = Units.degreesToRadians(canCoder.getAbsolutePosition().getValue()) - Units.rotationsToRadians(encoderOffset);
+            inputs.absoluteEncoderPosition = Units.degreesToRadians(canCoder.getAbsolutePosition()) - Units.rotationsToRadians(encoderOffset);
         }
-        inputs.isAbsoluteEncoderConnected = canCoder.getVersionMinor().getValue() != -1;
+        inputs.isAbsoluteEncoderConnected = canCoder.getFirmwareVersion() != -1;
```

Comment:
Is this the correct way to check if the encoder is connected?

Reply:
yes, assaf did it also last year and it worked